src/interfaces/navigation.ts

```diff
@@ -1,64 +1,71 @@
 /**
- * Описание конечного пункта в меню навигации.
+ * Description of the destination in the navigation menu.
  *
- * @param viewName Имя вью.
+ * @param viewName Identifier of view.
  */
 export interface ViewNavigationItem {
-    viewName: string
+    /** TODO identifier will be nullable in next major */
+    id?: string
+    viewName: string,
+    hidden?: boolean,
 }
 
 /**
- * TODO
+ * Returns MenuItem if it is ViewNavigationItem
  * 
- * @param item 
+ * @param item to be identified as view
  */
-export function isViewNavigationItem(item: any): item is ViewNavigationItem {
+export function isViewNavigationItem(item: MenuItem): item is ViewNavigationItem {
     return !!item && ('viewName' in item)
 }
 
 /**
- * Описание категории в меню навигации.
- *
- * Используется для создания уровней вложенности пунктов меню.
- *
- * @param categoryName Название категории.
- * @param child Список категорий или пунктов меню, входящих в категорию.
+ * Description of the category in the navigation menu.
+ * Used to create nesting levels of menu items.
+ * @param categoryName The name of the category.
+ * @param child list of categories or menu items included in a category.
+ * TODO Deprecated. ViewNavigationCategory will be deleted in next major
  */
 export interface ViewNavigationCategory {
     categoryName: string,
     child: Array<ViewNavigationCategory | ViewNavigationItem>
 }
 
 /**
- * 
- * @param item 
+ * @param item
+ * TODO Deprecated. ViewNavigationCategory will be deleted in next major
  */
 export function isViewNavigationCategory(item: any): item is ViewNavigationCategory {
     return !!item && ('categoryName' in item)
 }
 
 /**
- * Описание группы в меню навигации.
+ * Description of groups in the navigation menu.
  *
- * Используется как корневой элемент, включающий в себя категории и пункты меню.
+ * Used to create nesting levels of menu items.
  *
- * @param title Имя группы.
- * @param child Список категорий или пунктов меню, входящих в группу.
+ * @param title Title of group. Navigation element shows it to user.
+ * @param child Array of navigation elements specified below group(View or inner Group)
  */
 export interface ViewNavigationGroup {
-    title: string
-    child: Array<ViewNavigationItem | ViewNavigationCategory>
+    /** TODO identifier will be nullable in next major */
```

Comment:
It'll be easier to find all planned TODOs if they'll mention target version directly:
```tsx
/** TODO identifier will be nullable in 2.0.0 */
```

Reply:
Fixed.